doc/source/whats_new.rst

```diff
@@ -137,13 +139,13 @@ API
    - find_events and read_events functions have a new parameter `mask` to set some bits to a don't care state by `Teon Brooks`_
 
    - New channels module including layouts, electrode montages, and neighbor definitions of sensors which deprecates
-    ``mne.layouts`` by `Denis Engemann`_
+    `mne.layouts` by `Denis Engemann`_
```

Comment:
you should build docs from the `mne/doc` directory. Did you try adding a line to the `Makefile` with appropriate arguments such that you can just do `make html_dev-noflow` or whatever our naming convention is?